### PR TITLE
Refactoring - Separated into files

### PR DESCRIPTION
**Pull request by TylersGit**
Quite a big overhaul of the organization.

I tried to change as little code as possible and focused on separating main.py into more manageable files.
I will send you the current .env file, which seems to work so far with the program.

From my manual testing nothing is broken.
Let me know if anything needs to be fixed before merging.

### CODE REVIEW

**Comment by TylersGit** [comment]
Rebased the last 11 commits for a cleaner repo history

File: bin/main_GUI.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+create_ui_button("Renew SFTP", 6, sftp_renew_button)
+create_ui_button("Open WinSCP", 7, open_winscp_button)
+
+# Constantly keeps processing buttons
+def process_buttons_loop():
+    buttons_process()
+    root.after(1, process_buttons_loop)
+
+# Starts button process loop
+process_buttons_loop()
```
Comment (StamLord) [inline bin/main_GUI.py:125]: Maybe we should add Start() function here that will call both "process_buttons_loop" and "root.mainloop()" so we can choose when to start everything from outside

Comment (TylersGit) [reply bin/main_GUI.py:125]: We can do that. 
I'll create an issue for it, and handle it in a separate pull request. 

This one is big enough as it is.

File: main.py [hunk 2]
```diff
@@ -1,348 +1,40 @@
...
-wd_event_handler.on_modified= on_created
+# WinSCP config
+# Should be deleted after refactoring SFTP operations
+winscp_path = os.environ.get("WINSCP_PATH")
+sftp_url = os.environ.get("SFTP_URL")
+sftp_user = os.environ.get("SFTP_USER")
+sftp_pass = os.environ.get("SFTP_PASS")
 
-wd_observer = Observer()
 wd_observer.schedule(wd_event_handler, downloads_path, recursive=False)
```
Comment (StamLord) [inline main.py:37]: Same as with main_gui - we should consider adding a StartWatchdog() function inside watchdog.py so we can call it and it here will handle starting the observer. That way we don't need to worry about event_handler / observer and treat it as a black box

**Comment by TylersGit** [comment]
I plan to add setup functions both to the GUI and to Watchdog, but in a separate branch.

**Review by StamLord — APPROVED** [review]
Nice work. A bit too divided for my taste but that's ok. I would have sftp_operations and os_operations under just "operations" for example

Also I think if we go this route we can add Start or Initialize functions to both gui and watchdog modules. That way we can call these functions and they will handle everything without main.py worrying about what each module needs or does.

**Pull request merged**